### PR TITLE
Fix examples & enums

### PR DESCRIPTION
**Pull request by dirtycajunrice**
## Changes

- Linear: https://linear.app/thirdweb/issue/INF-171/
- Here is the problem being solved.
- Here are the changes made.

## How this PR will be tested

- [ ] Open the dashboard and click X. Result: A modal should appear.
- [ ] Call the /foo/bar API. Result: Returns 200 with "baz" in the response body.

## Output

(Example: Screenshot/GIF for UI changes, cURL output for API changes)

---

## PR-Codex overview
This PR primarily focuses on code cleanup and refactoring, improving error handling, updating comments, and enhancing type definitions across various services and routes. It also includes modifications to schemas and responses to ensure better clarity and consistency.

### Detailed summary
- Added `.idea` and JetBrains-related files to `.gitignore`.
- Improved error handling by removing specific error variables in `catch` blocks.
- Changed `let` declarations to `const` where applicable for better immutability.
- Updated response schemas to use more precise types and structures.
- Modified example responses to reflect more accurate data types.
- Enhanced documentation comments for clarity and completeness.
- Refactored service methods to use more descriptive names (e.g., `getAll()` to `list()`).
- Added new `testWebhook` method in `WebhooksService` for testing webhook functionality.
- Updated package versions for ESLint and TypeScript-related packages.

> The following files were skipped due to too many changes: `sdk/src/services/ContractMetadataService.ts`, `sdk/src/services/ContractService.ts`, `src/tests/auth.test.ts`, `sdk/src/services/ContractSubscriptionsService.ts`, `sdk/src/services/AccountFactoryService.ts`, `sdk/src/services/ContractRolesService.ts`, `sdk/src/services/ContractRoyaltiesService.ts`, `sdk/src/services/ConfigurationService.ts`, `sdk/src/services/AccountService.ts`, `sdk/src/services/TransactionService.ts`, `sdk/src/services/MarketplaceOffersService.ts`, `sdk/src/services/MarketplaceEnglishAuctionsService.ts`, `yarn.lock`, `sdk/src/services/BackendWalletService.ts`, `sdk/src/services/MarketplaceDirectListingsService.ts`, `sdk/src/services/DeployService.ts`, `sdk/src/services/Erc20Service.ts`, `sdk/src/services/Erc721Service.ts`, `sdk/src/services/Erc1155Service.ts`, `sdk/old_openapi.json`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by dirtycajunrice** [comment]
* **#778** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/engine/778?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/thirdweb-dev/engine/778?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

**Review by d4mr — CHANGES_REQUESTED** [review]
would need to make a pass, remove es-lint related stuff and ignore autogen sdk code for formatter to get a more usable diff

File: eslint.config.mjs
[the remark this answers is not in the document]
Comment (dirtycajunrice) [reply eslint.config.mjs:1]: I didnt add anything, i just migrated the existing config that already existed to one that didnt break.

**Comment by arcoraven** [comment]
I'm pulling pieces from this PR and merging them into main. I won't reopen this unless you want to rebase it (probably more effort than it's worth).

**Pull request closed without merging**